src/pyasm/security/security.py

```diff
@@ -1262,6 +1262,7 @@ def login_as_guest(my):
 
         my._do_login()
 
```

Comment:
What is this block of code for exactly? Should we just delete it instead of commenting it out? Maybe there should be something about why it's commented out